docs/cluster.md

```diff
@@ -0,0 +1,43 @@
+# KBS Cluster
+
+KBS provides a simple cluster defined by `docker-compose`. 
```

Comment:
+ `including KBS itself, [Attestation-Service](), [RVPS]() and [CoCo Keyprovider]().`
And you can briefly describe the role of these components.

If we have a graph for this, that would be very nice!